docs/intro/install/cloud9.rst

```diff
@@ -26,11 +26,11 @@ Run the following in cloud9's terminal:
     rm -rf * # Clear out the c9 hello-world stuff
     composer create-project elgg/elgg:~1.11 . # the hotness
     cp install/config/htaccess.dist .htaccess
-    cp engine/settings.example.php engine/settings.php
+    cp engine/settings.example.php settings.php
```

Comment:
Or should it be moved to install/config/settings.example.php?? that is where htaccess.dist is also located